frappe/printing/doctype/letter_head/letter_head.py

```diff
@@ -13,8 +14,14 @@ def before_insert(self):
 
 	def validate(self):
 		self.set_image()
-		if not self.is_default:
-			if not frappe.db.sql("""select count(*) from `tabLetter Head` where ifnull(is_default,0)=1"""):
+		self.validate_disabled_and_default()
+
+	def validate_disabled_and_default(self):
+		if self.disabled and self.is_default:
+			frappe.throw(_("Letter Head cannot be both, {0} and {1}").format(frappe.bold("Disabled"), frappe.bold("Default")))
+		
+		if not self.is_default and not self.disabled:
+			if not frappe.db.sql("""select count(*) from `tabLetter Head` where ifnull(is_default,0)=1""")[0][0]:
```

Comment:
try and use the ORM where possible
`if not frappe.db.exists('Letter Head', dict(is_default=1)`